src/models/companyplan.js

```diff
@@ -0,0 +1,15 @@
+
+module.exports = (sequelize, DataTypes) => {
+  const CompanyPlan = sequelize.define('CompanyPlan', {
+    label: { type: DataTypes.STRING, allowNull: false }
+  }, {});
+  CompanyPlan.associate = (models) => {
+    CompanyPlan.hasMany(models.Company, {
+      as: 'companies',
+      foriegnKey: 'companyPlanId',
```

Comment:
```suggestion
      foreignKey: 'companyPlanId',
```

Reply:
Im seeing foriegnkey on others too, is this issue from sequelize?

Reply:
@Baystef .It's been corrected. Thanks for taking note.